src/net.h

```diff
@@ -315,14 +317,14 @@ class CConnman
 private:
     struct ListenSocket {
         SOCKET socket;
-        bool whitelisted;
+        CNetPermissionFlags permissions;
```

Comment:
nit with regard to coding style:
* Newly introduced classes don't have the `C` prefix
* Members start with `m_`, `m_snake_case`
* `if` statements are either in one line or the code blocks are wrapped in `{}`.

```suggestion
        NetPermissionFlags m_permissions;
```